tests/utils/trainer_fit.py

```diff
@@ -33,7 +32,6 @@ def get_total_loss(model: ComposerModel, dataloader: DataLoader, device: Device)
 
 
 def train_model(composer_trainer_hparams: TrainerHparams, max_epochs: int = 2, run_loss_check: bool = False):
```

Comment:
Do we still need this test given `test_trainer.py` ? Should we refactor this into that test file and its pattern?

Reply:
Nothing is using `train_model`....should I remove it?

Reply:
Yes